node/hyperbahn/hyperbahn.thrift

```diff
@@ -0,0 +1,16 @@
+exception Exception {
+    1: required string message
+}
+
+exception NoHost {
+    1: required string message
+}
+
+service Hyperbahn {
+    list<string> hostportsByService(
```

Comment:
This is not how you do thrift. Always take one argument; a struct. Always return one argument a struct.